### PR TITLE
[TD-13366]<docs>: update installation instructions of datasource plugin in tdinsight

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

File: dashboards/TDinsight.md [hunk 1]
```diff
@@ -195,9 +195,15 @@ For special use cases, `-O` would set the organization id when you use Grafana C
 Install the TDengine data-source plugin from GitHub.
```
Comment (sangshuduo) [inline dashboards/TDinsight.md:195]: For aligning with Chinese version, how about change to following:
Install the latest version of TDengine data-source plugin from GitHub.

**Review by sangshuduo — COMMENTED** [review]
see my comment

File: dashboards/TDinsight.md (hunk 1 above)
Comment (zitsen) [reply dashboards/TDinsight.md]: Changed.

**Review by huolibo — APPROVED** [review]

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**